openapi-core/src/main/java/io/ballerina/openapi/core/generators/service/model/OASServiceMetadata.java

```diff
@@ -31,13 +31,15 @@ public class OASServiceMetadata {
     private final boolean nullable;
     private final String license;
     private final boolean generateServiceType;
+    private final boolean generateWithoutDataBinding;
 
     private OASServiceMetadata(Builder clientConfigBuilder) {
         this.openAPI = clientConfigBuilder.openAPI;
         this.filters = clientConfigBuilder.filters;
         this.nullable = clientConfigBuilder.nullable;
         this.license = clientConfigBuilder.license;
         this.generateServiceType = clientConfigBuilder.generateServiceType;
+        this.generateWithoutDataBinding = clientConfigBuilder.generateWithoutDataBinding;
```

Comment:
```suggestion
        this.generateWithoutDataBinding = builder.generateWithoutDataBinding;
```
Shall we change the `clientConfigBuilder` into `builder` or `serviceMetaDataBuilder` something?

Reply:
Updated